### PR TITLE
Fix exception when frames.length isz ero

### PR DESCRIPTION
**Pull request by surfdude75**

### CODE REVIEW

**Comment by mrocajr** [comment]
LGTM

File: lib/mp4/transmuxer.js [hunk 1]
```diff
@@ -503,9 +503,9 @@ VideoSegmentStream = function(track) {
 
     // For the last frame, use the duration of the previous frame if we
     // have nothing better to go on
-    if (frames.length &&
+    if (frames.length && (
```
Comment (forbesjo) [inline lib/mp4/transmuxer.js]: Could you move this paren to the next line so that this can match the rest of the code in our plugins.

**Comment by forbesjo** [comment]
I'll make a new PR with this fix and add that one little change.

**Comment by imbcmdth** [comment]
Fixed in #90

**Pull request closed without merging**